src/elements/form/extras/address/Address.scss

```diff
@@ -3,6 +3,7 @@ novo-address {
     flex-flow: row wrap;
 
     input {
+        min-width: 10px;
```

Comment:
Seems like a hack? Is the overall way we are doing the flex css around the address incorrect?